website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -955,9 +955,10 @@ the actual processing logic. You don't need to manually extract the elements
 from the input collection; the Beam SDKs handle that for you. Your `process` method
 should accept an argument `element`, which is the input element, and return an
 iterable with its output values. You can accomplish this by emitting individual
-elements with `yield` statements. You can also use a `return` statement
-with an iterable, like a list or a generator. Don't mix `yield` and
-`return` statements in the same `process` method - this leads to [undefined behavior](https://github.com/apache/beam/issues/22969).
+elements with `yield` statements, and use `yield from` to emit all elements from
+an iterable, such as a list or a generator. Using `return` statement
+with an iterable is also acceptable as long as you don't mix `yield` and
```

Comment:
`yield` could cause the this error `cannot pickle 'generator' object` for some cases. This is why I did not update the doc in https://github.com/apache/beam/pull/25743

Reply:
as discussed offline `cannot pickle 'generator' object` was happening in different contexts (outside of `process` methods). LMK if you think we should merge this, feel free to suggest a different wording.

Reply:
Looks good. Thanks.